README.md

```diff
@@ -72,12 +72,19 @@ When adding support for a new object, or updating existing code, it can be usefu
 TrueNAS machine from time to time. In order to help do that easily, you can drop a `.auth.yaml` file in the root of
 the repository, with the following content:
 
-```
+```yaml
 host: "some.host.name"
 username: "someuser"
 password: "somepassword"
 ```
 
+Alternatively, an api key can be used:
+
+```yaml
+host: "some.host.name"
+api_key: "someapikey"
+```
+
```

Comment:
I think an API key is strictly the better option, so I'd prefer to document it first.  I might suggest some language like "if you must use a username and password..." for the old ways.

Reply:
I think making the api_key the suggested default is good.  If API key is not available for prior versions, it may not make sense to be as dogmatic about discouraging the username route.